meson.build

```diff
@@ -73,13 +73,14 @@ if not get_option('freetype').disabled()
   freetype_dep = dependency('freetype2', required: false)
 
   if not freetype_dep.found() and cpp.get_id() == 'msvc'
-    if cpp.has_header('ft2build.h')
-      freetype_dep = cpp.find_library('freetype', required: false)
-    endif
+    freetype_dep = cpp.find_library('freetype', required: false,
+                                    has_headers: ['ft2build.h'], )
   endif
 
-  if not freetype_dep.found() and get_option('freetype').enabled()
-    freetype_dep = dependency('freetype2', fallback: ['freetype2', 'freetype_dep'])
+  if not freetype_dep.found()
+    freetype_dep = dependency('freetype2', fallback: ['freetype2', 'freetype_dep'],
+                              default_options: ['harfbuzz=disabled'],
```

Comment:
just one thing, can we have yet one other attempt before going for tweaking default option? Just that let's have it last resort, and please rebase the change as I had to fork cairo wrap https://github.com/harfbuzz/harfbuzz/pull/2517/files/d5d15f985e9eb6ef67cb5b29811f1d7d1c50543a#diff-c8106e8bf415bce9acbd51c8f11eb092 to make it work.

Reply:
Commented on https://github.com/harfbuzz/harfbuzz/pull/2519